src/app/ASKS/ask/index.js

```diff
@@ -377,6 +380,21 @@ const Ask = ({
           config={config}
         />
       )}
+      {component === 'approve' && (
```

Comment:
I don't think approve is a valid dataType/component? If it is supposed to be a toggle with labels in it, then the component should be flag which handles our toggles.
Also, only standard generic form components as required are to be handled in ask file. Please don't add any specefic template related UI component in the file.

Reply:
Instead type can be used to further branch off the component inside Flag component. The name of the component should be LabeledFlag or something more semantic.

Reply:
Ok, using the component type or something else?

Reply:
Yeah type can be used or the component in itself can be called LabeledFlag or LabeledToggle or something that helps us distinguish that it is a toggle field with labels in it.

Reply:
I have now created the labelled_flag type, however still not getting the labels from the backend yet, not sure where to do that